### PR TITLE
Display all cards

### PR DESCRIPTION
**Pull request by LeahWinters**
# PR Description
Please include a summary of the change:

Completed by Melissa and Léah. Created a component for the cocktail card and connected it to the cocktail page. Added functionality so that the cards display on the cocktail page, and styled accordingly.

Screen Shot 2020-07-23 at 4 12 22 PM

Commit Emojis:
:art: CSS modifications/updates.
:crystal_ball: JS modifications/updates.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Tests (non-breaking change which adds testing)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [X] My code follows the Turing style guidelines
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/AllCocktailsPage/AllCocktailsPage.tsx [hunk 1]
```diff
@@ -19,33 +21,41 @@ const AllCocktailsPage: React.SFC = () => {
       idDrink: 'string'
 		}]
   });
+
   const [error, setError] = useState<string>("");
 
   const displayAllCocktails = async (): Promise<any> => {
```
Comment (asodemann18) [inline src/AllCocktailsPage/AllCocktailsPage.tsx:27]: Nice job typing the function

File: src/AllCocktailsPage/AllCocktailsPage.tsx [hunk 2]
```diff
@@ -19,33 +21,41 @@ const AllCocktailsPage: React.SFC = () => {
...
       setError(error.message);
     }
 	};
 
   useEffect(() => {
     displayAllCocktails();
-	}, []);
+  }, []);
+
+  const cocktailsArray = Object.values(allCocktails.drinks);
```
Comment (asodemann18) [inline src/AllCocktailsPage/AllCocktailsPage.tsx:40]: Mod 2 skills still going strong!

File: src/CocktailCard/CocktailCard.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './CocktailCard.scss';
+
+export interface CocktailsCardProps {
+  strDrink: string;
+  strDrinkThumb: string;
+  idDrink: string;
+}
+
+const CocktailCard: React.SFC<CocktailsCardProps> = (props) => {
```
Comment (asodemann18) [inline src/CocktailCard/CocktailCard.tsx:10]: Looks good!

**Review by asodemann18 — APPROVED** [review]
Looks great!

File: src/AllCocktailsPage/AllCocktailsPage.tsx [hunk 4]
```diff
@@ -19,33 +21,41 @@ const AllCocktailsPage: React.SFC = () => {
...
+  console.log('array', cocktailsArray);
+  
+  const cocktailCards = cocktailsArray.map(cocktail => {
+    return (
+      <CocktailCard 
+        strDrink={cocktail.strDrink}
+        strDrinkThumb={cocktail.strDrinkThumb}
+        idDrink={cocktail.idDrink}
+      />
+    );
```
Comment (st0nebreaker) [inline src/AllCocktailsPage/AllCocktailsPage.tsx:51]: Beautiful logic!

File: src/AllCocktailsPage/AllCocktailsPage.tsx [hunk 5]
```diff
@@ -19,33 +21,41 @@ const AllCocktailsPage: React.SFC = () => {
...
     }
 	};
 
   useEffect(() => {
     displayAllCocktails();
-	}, []);
+  }, []);
+
+  const cocktailsArray = Object.values(allCocktails.drinks);
+
```
Comment (st0nebreaker) [inline src/AllCocktailsPage/AllCocktailsPage.tsx:41]: Nice use of Object.values 😜  @LeahWinters @Melizzo

File: src/CocktailCard/CocktailCard.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <section className="cocktail-card">
+      <h3 className="card-title">{props.strDrink}</h3>
+      <img className='CC-img' src={props.strDrinkThumb} alt={props.strDrink} />
+    </section>
+  )
+}
+
+export default CocktailCard;
```
Comment (st0nebreaker) [inline src/CocktailCard/CocktailCard.tsx:19]: such modular code with these cocktail cards!

**Comment by st0nebreaker** [comment]
Beautiful cards, Leah! @LeahWinters

**Pull request merged**